src/ConstLinearLayer.jl

```diff
@@ -2,17 +2,11 @@ import ChainRulesCore: rrule
 using LuxCore
 using LuxCore: AbstractExplicitLayer
 
-struct ConstLinearLayer{T} <: AbstractExplicitLayer # where {in_dim,out_dim,T}
-    W::AbstractMatrix{T}  
-    position::Union{Vector{Int64}, UnitRange{Int64}}
-    in_dim::Integer
-    out_dim::Integer
+struct ConstLinearLayer <: AbstractExplicitLayer
+    op
```

Comment:
this gives you a type instability. you still need to specify the type of the operator.  Something like this: 
```julia
struct ConstLinL{TOP} 
    op::TOP
end
```